src/networks.json

```diff
@@ -580,6 +580,20 @@
     "explorer": "https://polygonscan.com",
     "start": 9834491,
     "logo": "ipfs://QmaGokGqgjknfa4xnXKnnwC5ZyXzUjQ7p6KEe4D8G5uFFE"
+  },  
+  "188": {
+    "key": "188",
+    "name": "Bytom Sidechain",
+    "shortName": "BMC",
+    "chainId": 188,
+    "network": "mainnet",
+    "multicall": "0x10cd4fa0D02eD0f33e5b569C371194F39898Ff3b",
+    "rpc": [
+      "https://mainnet.bmcchain.com/"
```

Comment:
Looks like this is not a full archive node @MartinBTM 
It give me `missing trie node c5bd7f02c614571ed4ff4693e2ed3801391ad5eb9fb3d88d54be94fed8193633 (path` error if i try older block

Reply:
Also something wrong with multicall, it give this error everytime
```
Transaction reverted without a reason string [
```

Reply:
sure，Thanks， we will check it

Reply:
we have changed to archive mode now, and deploy a new mutil call, please check the pr ,Thanks very much